src/stack_aggregator.py

```diff
@@ -430,7 +430,10 @@ def remove_duplicate_cve_data(epv_list):
             graph_dict[pv]['cves'] = list()
         if 'cve' in data:
             cve = graph_dict[pv].pop('cve')
-            graph_dict[pv]['cves'].append(cve)
+            # Fixes Issue
+            # https://github.com/fabric8-analytics/fabric8-analytics-vscode-extension/issues/328
```

Comment:
@miteshvp , it would be good to have the issue details in the commit message rather than in the source code.

Reply:
@arajkumar - It's already mentioned in commit message.